README.md

```diff
@@ -15,8 +15,19 @@
     </a>
 </p>
 
-`connqc` is a tool to control the quality of the network transport between multiple regions.
-It consists of a server that listens for TCP probing messages, and a client that sends said messages.
+`connqc` is a tool to check the quality of the network transport between networked hosts.
+We have, in our past encountered the following problems in the wild between two hosts:
```

Comment:
`our past` is a little odd. Also running directly into `We have` and `our` does not read well.

```suggestion
It has been designed to detect the connection issues between two hosts that have been observed in the wild, such as:
```

Reply:
I'm not fond of using `in the wild` here 😄 I'd rather go more corporate with something like

```suggestion
Known connection issues that `connqc` has been designed to detect include:
```

Here _known_ implies commonality, which conveys IMO the same feeling, while keeping a more technical/professional tone.